README.md

```diff
@@ -91,13 +91,13 @@ You can use **navi** as a widget to your shell. This way, your history is correc
 In order to use it, add this line to your `.bashrc`-like file:
 ```sh
 # bash
-source "$(navi widget bash)"
+source <(navi widget bash)
```

Comment:
I'm not super versed in bash/zsh/fish/rust, but I couldn't find the reason in the code that required this change.

AFAIK `source` needs a path to, well, source in the current shell.
This, [if I understand it correctly](http://www.gnu.org/software/bash/manual/bash.html#Process-Substitution), reads from the given file and passes it as the stdin for `source`, which might work but I think is unnecessary.

Reply:
Previously the command returned a path to a shell script. Now it prints the content of the shell script.